packages/did-auth-siop-op-authenticator/src/session/OperatingPartySession.ts

```diff
@@ -0,0 +1,231 @@
+import { DIDDocumentSection, IIdentifier } from '@veramo/core'
+import { _ExtendedIKey, mapIdentifierKeysToDoc } from '@veramo/utils'
+import { OP, PresentationExchange } from '@sphereon/did-auth-siop/dist/main'
+import { SubmissionRequirementMatch, VerifiableCredential } from '@sphereon/pe-js'
+import { parseDid } from '../utils';
+import {
+  PassBy,
+  ResponseMode,
+  ParsedAuthenticationRequestURI,
+  PresentationDefinitionWithLocation,
+  VerifiedAuthenticationRequestWithJWT,
+  VerifyAuthenticationRequestOpts,
+  VerifiablePresentationTypeFormat,
+  VerificationMode,
+} from '@sphereon/did-auth-siop/dist/main/types/SIOP.types'
+import {
+  IOperatingPartySessionArgs,
+  IOpsAuthenticateWithDidSiopArgs,
+  IOpsGetDidSiopAuthenticationRequestDetailsArgs,
+  IOpsGetDidSiopAuthenticationRequestFromRpArgs,
+  IOpsSendDidSiopAuthenticationResponseArgs,
+  IOpsVerifyDidSiopAuthenticationRequestUriArgs,
+  IAuthRequestDetails,
+  IMatchedPresentationDefinition,
+  IRequiredContext,
+} from '../types/IDidAuthSiopOpAuthenticator'
+
+const fetch = require('cross-fetch')
+
+export class OperatingPartySession {
+  public readonly id: string
+  public readonly identifier: IIdentifier
+  public readonly didMethod: string
+  public readonly section: DIDDocumentSection
```

Comment:
didDocSection or verificationMethodSection?

Reply:
renamed to verificationMethodSection